src/arch/mips/reg/mips.rs

```diff
@@ -1,5 +1,6 @@
 use crate::arch::Registers;
 use crate::internal::LeBytes;
+use core::convert::TryInto;
 
 use num_traits::PrimInt;
```

Comment:
nit: I just realized the imports should be shuffled around a bit:

```suggestion
use core::convert::TryInto;

use num_traits::PrimInt;

use crate::arch::Registers;
use crate::internal::LeBytes;
```